ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -4766,11 +4766,8 @@ let host_all_editions printer rpc session_id params =
 
 let host_evacuate printer rpc session_id params =
   let network =
-    match List.assoc_opt "network-uuid" params with
-    | None ->
-        Ref.null
-    | Some uuid ->
-        Client.Network.get_by_uuid rpc session_id uuid
+    List.assoc_opt "network-uuid" params
+    |> Option.fold ~none:Ref.null ~some:Ref.of_string
```

Comment:
This looks wrong, shouldn't this be 
```suggestion
    |> Option.fold ~none:Ref.null ~some:(Client.Network.get_by_uuid rpc session_id)
```
instead?

Reply:
Yes! Good catch ^^'